diff --git a/node_modules/hive-tx/helpers/call.js b/node_modules/hive-tx/helpers/call.js
index 8727036..c9978bb 100644
--- a/node_modules/hive-tx/helpers/call.js
+++ b/node_modules/hive-tx/helpers/call.js
@@ -8,30 +8,32 @@ const config = require('../config')
  * @param {Number}timeout - optional - default 10 seconds
  */
 const call = async (method, params = [], timeout = 10) => {
-  let resolved = 0
+  let resolved = false
+
   return new Promise((resolve, reject) => {
-    axios
-      .post(
-        config.node,
-        JSON.stringify({
-          jsonrpc: '2.0',
-          method,
-          params,
-          id: 1
-        })
-      )
-      .then(res => {
-        if (res && res.status === 200) {
-          resolved = 1
-          resolve(res.data)
-        }
-      })
+    fetch(config.node, {
+      method: 'POST',
+      body: JSON.stringify({
+        jsonrpc: '2.0',
+        method,
+        params,
+        id: 1
+      }),
+      headers: { 'Content-Type': 'application/json' }
+    })
+    .then(res => {
+      if (res && res.status === 200) {
+        resolved = true
+        resolve(res.json())
+      }
+    })
+
     setTimeout(() => {
       if (!resolved) {
         reject(new Error('Network timeout.'))
       }
     }, timeout * 1000)
-  })
+  });
 }
 
 module.exports = call
