diff --git a/node_modules/hive-tx/index.d.ts b/node_modules/hive-tx/index.d.ts
index 7286729..b4beada 100644
--- a/node_modules/hive-tx/index.d.ts
+++ b/node_modules/hive-tx/index.d.ts
@@ -1,4 +1,4 @@
-import { PrivateKey as PK } from './helpers/PrivateKey'
+import { PrivateKey as PK } from './helpers/PrivateKey';
 
 declare module 'hive-tx'
 
@@ -46,7 +46,7 @@ export const config: {
   node: string
 }
 
-export function call(method: string, params?: any[], timeout?: number): any
+export function call(method: string, params?: any, timeout?: number): any
 
 /** Don't need anymore - deprecated */
 export function updateOperations(): void
